src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -3,85 +3,64 @@ title: Proof-of-stake (PoS)
 description: An explanation of the proof-of-stake consensus protocol and its role in Ethereum.
 lang: en
 sidebar: true
-incomplete: true
+incomplete: false
 ---
 
-Ethereum is moving to a consensus mechanism called proof-of-stake (PoS) from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/). This was always the plan as it's a key part in the community's strategy to scale Ethereum via [upgrades](/upgrades/). However getting PoS right is a big technical challenge and not as straightforward as using PoW to reach consensus across the network.
+Ethereum is moving to a consensus mechanism called proof-of-stake (PoS) from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/). This was always the plan because PoS is demonstrably more secure than PoW, uses drastically less energy, and enables new scaling solutions to be implemented. However, proof-of-stake is also more complex than proof-of-work. Refining the proof-of-stake mechanism has taken years of research and development, and the challenge now is implementing it on the live Ethereum network - a process known as ["The Merge"](/upgrades/merge/).
```

Comment:
> demonstrably more secure than PoW

*If implemented correctly. Would we want to say this has been demonstrated already given that we still haven't undergone the merge? Or should we note it as theoretical?

Reply:
ok, changed "is demonstrably" to "is thought to be"